build.sbt

```diff
@@ -1,6 +1,4 @@
-scalaVersion := "2.10.6"
-
-crossScalaVersions := Seq("2.10.6", "2.11.8")
+scalaVersion in ThisBuild := "2.11.8"
```

Comment:
Hmm, I would also be fine with not supporting Scala 2.12 just yet, if it's a bunch of extra work. We should do it eventually but I guess it doesn't have to be now. Nobody has asked for it yet that I am aware of.

I'll leave it up to you. If you _do_ want to add 2.12 then yeah, I think it makes sense to do that all as part of #220.